hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OMDBUpdateEvent.java

```diff
@@ -107,7 +107,7 @@ OMUpdateEventBuilder setSequenceNumber(long sequenceNumber) {
      * @return OMDBUpdateEvent
      */
     public OMDBUpdateEvent build() {
-      return new OMDBUpdateEvent<KEY, VALUE>(
+      return new OMDBUpdateEvent<K1, V1>(
```

Comment:
Can we leave the Generic Types to actual KEY, VALUE instead of K1, V1. I feel it is more readable than single letters.

Reply:
Thanks for the review, done.